packages/paste-website/src/pages/components/topbar/index.mdx

```diff
@@ -0,0 +1,207 @@
+      },
+      navigationData,
+    },
+  };
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/topbar"
+  storybookUrl="/?path=/story/components-topbar--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+## Guidelines
+
+### About Topbar
+
+A Topbar is sticky to the top of the screen and includes application-wide controls including User Dialog (coming soon!), [Product Switcher](#product-switcher), [Account Switcher](#account-switcher), and search [Input](/components/input).
+
+## Examples
+
+### Default Topbar
+
+Use a Topbar at the top of your application to house application-wide controls.
+
+Account-specific controls are aligned to the left (e.g. [Account Switcher](#account-switcher), account type indicator [Badges](/components/badge), and possible upgrade actions).
+
+User-specific controls are aligned to the right (e.g. search [Input](/components/input), support [Button](/components/button), [Product Switcher](#product-switcher), and User Dialog (coming soon!)).
+
+<LivePreview scope={{Topbar}} noInline>
+  {defaultTopbarExample}
+</LivePreview>
+
+## Account Switcher
+
+An Account Switcher is a stylized [Menu Badge](/components/menu#menu-badge) with a list of actions related to a user’s accounts.
+
+Examples of account-related actions include navigating to other recently visited accounts, creating an account, and viewing all of a user’s accounts. Other names for accounts in Twilio products include workspaces and projects.
+
+### Accessibility
+
+Refer to the accessibility section of [Menu](/components/menu).
+
+### Examples
+
+Use an Account Switcher to display actions that a user can trigger related to their accounts.
+
+<LivePreview
+  scope={{
+    useMenuState,
+    Menu,
+    MenuBadge,
+    MenuItem,
+  }}
+  noInline
+  showOverflow
+>
+  {defaultAccountSwitcherExample}
+</LivePreview>
+
+## User Dialog
+
+Coming soon!
+
+## Product Switcher
+
+A Product Switcher is an opinionated menu that presents a list of Twilio products that a user has access to and can switch between.
+
+### Accessibility
+
+Refer to the accessibility section of [Menu](/components/menu).
+
+### Examples
+
+Use a Product Switcher for when a user has access to multiple Twilio products so a user can switch between them.
+
+The default state of a Product Switcher item indicates that the control is static or not selected. A Product Switcher item can be placed into a selected state in the same way as a radio button. Use the `checked` prop to set a selected item by default.
+
+<LivePreview
+  scope={{
+    ProductSwitcher,
+    ProductSwitcherButton,
+    ProductSwitcherItem,
+    useProductSwitcherState,
+    LogoTwilioIcon,
+    ProductSegmentIcon,
+    ProductFlexIcon,
+    ProductEmailAPIIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {defaultProductSwitcherExample}
+</LivePreview>
```

Comment:
todo: swap this out for the Product Switcher example in PR https://github.com/twilio-labs/paste/pull/3255